content/blog/2023/03/10/2023-03-10-jenkins-newsletter.adoc

```diff
@@ -0,0 +1,149 @@
+
+=== FOSDEM 2023
+
+Returning to FOSDEM for the first in-person event since COVID was both exciting and nostalgic for our Jenkins contributors.
+It was exciting to see the same crowd size, and enthusiasm by attendees.
+Many thanks to the wonderful FOSDEM organizers for yet another fantastic event!
+
+image:/images/post-images/2023/02/21/2023-02-21-thoughts-on-FOSDEM-2023/image15.png[image,width=624,height=469]
+
+image:/images/post-images/2023/02/21/2023-02-21-thoughts-on-FOSDEM-2023/image13.png[image,width=624,height=469]
+
+=== Jenkins in Google Summer of Code (GSoC)
+
+We are thrilled to have been accepted to the Google Summer of Code 2023!!
+This will be Jenkins' eight (8th) year in participation with the program.
+Weekly GSoC office hours have begun as well - office hours are every Thursday @ 16:00 UTC (see link:/events/[Event Calendar] for details).
+If you missed the initial meetings, the recordings are below:
+
+- https://youtu.be/1glzTzCkMvw[February 23, 2023]
+- https://youtu.be/vqMHdAUy85U[February 28, 2023]
+
+Join in on all GSoC discussions on link:https://app.gitter.im/#/room/#jenkinsci_gsoc-sig:gitter.im[gitter]
+
+image:/images/post-images/2023/03/10/2023-03-10-jenkins-newsletter/image3.png[image,width=624,height=360]
+
+=== Jenkins Awards
+
+Award season is here! Nomination is closed but voting is https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform[OPEN].
+Congratulations to all nominees and thank you for your contributions! +
+See post about link:/blog/2023/02/23/cdf-awards/[the Jenkins awards].
+
+image:/images/post-images/2023/03/10/2023-03-10-jenkins-newsletter/image4.png[image,width=624,height=326]
+
+[[infrastructure]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/infrastructure.png[Infrastructure Update]
+Contributed by: author:dduportal[Damien Duportal]
+
+Following the FOSDEM where most of the (infra) team was present physically, February was a busy month for the Jenkins Infrastructure:
+
+- JFrog (repo.jenkins-ci.org) bandwidth reduction: Jenkins continuous integration jobs are now using significantly less internet bandwidth thanks to the artifact caching proxy implemented by the team.
+The artifact caching proxy is implemented on our three cloud providers so that artifacts can be downloaded from a local cache rather than accessing the artifact repository.
+- Jenkins LTS `2.375.3` is used on all of our controllers less than 3 days after its release
+- All Jenkins deprecated plugins had been removed on all of our controllers (jquery, momentjs, ace-editor)
+- All of our (six) Kubernetes clusters were upgraded from `1.23` to `1.24` baselines, in the three cloud providers
+- All of the private and internal web services are using TLS with certificates provided by Let's Encrypt (with Azure DNS challenge)
+- We contributed to Docker documentation after catching issues with Docker CE `23.x` fresh release and Docker BuildX (on Ubuntu)
```

Comment:
```suggestion
* In an effort to reduce bandwidth with JFrog (repo.jenkins-ci.org), Jenkins continuous integration jobs are now using significantly less internet bandwidth thanks to the artifact caching proxy implemented by the team.
The artifact caching proxy is implemented on our three cloud providers, so that artifacts can be downloaded from a local cache rather than accessing the artifact repository.
* Jenkins LTS `2.375.3` is now used on all of our controllers, less than 3 days after its release.
* We have removed all Jenkins deprecated plugins on all of our controllers such as jquery, momentjs, and ace-editor.
* We upgraded all six of our Kubernetes clusters from the `1.23` to `1.24` baseline in the three cloud providers.
* All of the private and internal web services are now using TLS with certificates provided by Let's Encrypt, along with Azure DNS challenge.
* We contributed to Docker documentation after catching issues with the Docker CE `23.x` fresh release and Docker BuildX on Ubuntu.
```
suggestion for wording/formatting of some sentences, update - to * for lists